publication-commons/src/main/java/no/unit/nva/publication/model/business/PublishingRequestCase.java

```diff
@@ -64,9 +62,11 @@ public static PublishingRequestCase createOpeningCaseObject(Publication publicat
         openingCaseObject.setStatus(TicketStatus.PENDING);
         openingCaseObject.setViewedBy(ViewedBy.addAll(openingCaseObject.getOwner()));
         openingCaseObject.setPublicationDetails(PublicationDetails.create(publication));
+        openingCaseObject.setWorkflow(PublicationWorkflow.REGISTRATOR_PUBLISHES_METADATA_AND_FILES);
```

Comment:
It was due to lack of information (needs a customer lookup query, which cannot be done in a constructor)
And it would keep existing tests green.
I added UNSET later, and that would be a more sensible value to assign at this step.